.travis.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       osx_image: xcode11
       script:
         - npm run test:example
-        - npm run test:ios
+        - travis_wait npm run test:ios
```

Comment:
Sometimes installing pods takes a bit over 10 minutes to fetch repos without any output. Travis thinks that build is stuck and stops it. `travis_wait` fixes this by outputting something to stdout every minute for 20 minutes or until script outputs something itself. Unfortunately it also hides output until the script returns exit code.